### PR TITLE
[kyber] Add more param sets, add 90s variant

### PR DESCRIPTION
**Pull request by dingens**

### CODE REVIEW

**Comment by franziskuskiefer** [comment]
This PR is stale. Can you update?

**Comment by dingens** [comment]
I noted I had taken all numbers from round 1. Now it should be correct. Can you have a thorough look?

**Comment by dingens** [comment]
I noted I had taken all numbers from round 1. Now it should be correct. Can you have a thorough look?

File: schema/paramset.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
                     "minimum": 1
                 },
                 "comment": {
-                    "type": "integer"
+                    "type": "string"
```
Comment (franziskuskiefer) [inline schema/paramset.json:70]: This is a schema change and shouldn't be in here.

File: encryption/kyber/ccakem/param/512.yaml [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+failure probability: -178
 number of operations: inf
 sizes:
     sk: 1632
-    pk: 736
-    ct|sig: 800
+    pk: 800
+    ct|sig: 736
     msg: 32
+    comment: "`sk` can be expanded from 32 byte seed"
```
Comment (franziskuskiefer) [inline encryption/kyber/ccakem/param/512.yaml:13]: "Finally, as all randomness in seeds, it is also possible to only store these seeds and re-run key generation during decapsulation."

**Comment by dingens** [comment]
> Can you add a link somewhere where you got the benchmarks from?
they're from their submission pdf, round 2 update ("Algorithm Specifications And Supporting Documentation (version 2.0)")

> Looks fine otherwise with the schema change moved out of here and the nit (when I was able to very the benchmark numbers).
I don't really get why the schema change is still here, I had already applied it in master. Maybe because of the fuckup. i'll merge master back in tomorrow and we'll see then

File: encryption/kyber/ccakem/param/512.yaml (hunk 2 above)
Comment (dingens) [reply encryption/kyber/ccakem/param/512.yaml:13]: Right, but what do you want to say by this? Do you assume this is possible for all schemes (is it?) and we should remove this info?

**Pull request merged**